modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/validator/AMImageValidatorImpl.java

```diff
@@ -54,6 +54,15 @@ public boolean isProcessingSupported(FileVersion fileVersion) {
 		return true;
 	}
 
+	@Override
+	public boolean isProcessingSupported(String mimeType) {
+		if (ContentTypes.IMAGE_SVG_XML.equalsIgnoreCase(mimeType)) {
```

Comment:
Can you reverse the order to: `mimeType.equalsIgnoreCase(ContentTypes....)`

We prefer things to fail fast if the mimeType is null.